tests/e2e/parallel/main_test.go

```diff
@@ -0,0 +1,360 @@
+package parallel_e2e
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	compv1alpha1 "github.com/ComplianceAsCode/compliance-operator/pkg/apis/compliance/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/ComplianceAsCode/compliance-operator/tests/e2e/framework"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var brokenContentImagePath string
+
+func TestMain(m *testing.M) {
+	f := framework.NewFramework()
+	err := f.SetUp()
```

Comment:
Do we use the same e2e pool for all the test?

Reply:
From the perspective or parallel tests, yes I believe so.